### ISSUE TITLE
unicode-bom rule isn't detected and causes entire file to not be reported

### ISSUE THREAD

**Issue by xt0rted** [xt0rted/stylelint-problem-matcher#1]
The `unicode-bom` rule doesn't output a line or column. This is causing the problem matcher to not detect it, and since it's the first rule in the results it's preventing the rest of the issues for the given file from being detected.

```console
scss/_test.scss
        ×  Unexpected Unicode BOM   unicode-bom        
 11:16  ×  Unexpected unit          length-zero-no-unit
 11:28  ×  Unexpected unit          length-zero-no-unit
 16:14  ×  Unexpected unit          length-zero-no-unit
 21:15  ×  Unexpected unit          length-zero-no-unit
 23:14  ×  Unexpected unit          length-zero-no-unit
```
